docker-compose.production.yml

```diff
@@ -82,6 +82,20 @@ services:
     secrets:
       - petabox_seed
 
+  solr_updater:
```

Comment:
This should also exist in `docker-compose.yml` since it's need in the local environment